content/en/docs/pipelines/multi-user.md

```diff
@@ -85,6 +75,12 @@ print(client.list_runs(experiment_id='<Your experiment ID>'))
 print(client.list_runs(namespace='<Your namespace>'))
 ```
 
+To store your user namespace as the default context, use the
+[`set_user_namespace`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.set_user_namespace)
+method. This method stores your user namespace in a configuration file at
+`$HOME/.config/kfp/context.json`. After setting a default namespace, the SDK
+methods default to use this namespace if no namespace argument is provided.
+
 We also provide a helper method that saves a user namespace as default context
```

Comment:
Please remove the following, since the added paragraph covers the same content:

We also provide a helper method that saves a user namespace as default context
in config file `$HOME/.config/kfp/context.json`. After setting a default
namespace, the SDK methods default to use this namespace if no `namespace`
argument provided.

Reply:
Thanks for catching this mistake!